mirrord/layer/src/lib.rs

```diff
@@ -221,20 +221,13 @@ pub(crate) static REMOTE_UNIX_STREAMS: OnceLock<Option<RegexSet>> = OnceLock::ne
 /// When true, localhost connections will stay local (won't go to the remote pod localhost)
 pub(crate) static OUTGOING_IGNORE_LOCALHOST: OnceLock<bool> = OnceLock::new();
 
-/// Tells us if the user enabled wants to ignore listening on localhost in [`IncomingConfig`].
-///
-/// ## Usage
-///
-/// When true, localhost connections will stay local - wont mirror or steal.
-pub(crate) static INCOMING_IGNORE_LOCALHOST: OnceLock<bool> = OnceLock::new();
+/// Incoming config [`IncomingConfig`].
+pub(crate) static INCOMING_CONFIG: OnceLock<IncomingConfig> = OnceLock::new();
```

Comment:
Why'd you remove ignore_localhost/ports?

Reply:
Instead of having a configuration variable for each I just have one variable (`INCOMING_CONFIG`) that has all of those fields.